polymesh_schema.json

```diff
@@ -1444,7 +1444,7 @@
         "params": [
           {
             "name": "from_custodian",
-            "type": "Option<IdentityId>",
```

Comment:
This was causing confusion on the UI side.  Since `isOptional` is false and the type is `Option<>`.

Reply:
No.  This was correct.  The type needs to be `Option<IdentityId>` and `isOptional: false`.

How were you testing this with the UI?

These RPC endpoints are used here:
https://github.com/PolymeshAssociation/polymesh-types/blob/1e6d8dd190d47354ff527c77edad59d3709ce070/src/rpc/index.ts#L279